how-to-guides/create-delete-projects/scripts/create_delete_projects.py

```diff
@@ -1,18 +1,16 @@
 from neptune import management
 
-PROJECT_NAME= "<WORKSPACE/PROJECT>"
-API_TOKEN= "<YOUR_API_TOKEN>"
```

Comment:
Same here,

overall for all contents in the "create-delete-projects" we wanted to have "you api token"

Reply:
I've reverted this, but don't you think we should let users pass arguments to scripts during runtime instead of expecting them to change these from within the script before execution? not sure how
 it'll be handled by github tests though

Reply:
In this case we need to have `YOUR_API_TOKEN` instead of `ANONYMOUS`. Users will pass their args anyways in place of these placeholders.

I seems that improvement in this area will come with getpass(), but let's keep that as a separate topic.